testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinPluginSmokeTest.groovy

```diff
@@ -130,36 +125,10 @@ class KotlinPluginSmokeTest extends AbstractKotlinPluginSmokeTest {
         version << TestedVersions.kotlin.versions
     }
 
-    def 'kotlin javascript (kotlin=#version, workers=#parallelTasksInProject)'() {
-
-        setupForKotlinVersion(version)
-
-        // kotlinjs has been removed in Kotlin 1.7 in favor of kotlin-mpp
-        assumeTrue(kotlinPluginVersion.baseVersion < VersionNumber.version(1, 7))
-
-        given:
-        useSample("kotlin-js-sample")
```

Comment:
❓ Should we remove the sample as well? If not, we should probably note that it works only with older versions of Kotlin/Gradle.

Reply:
Good catch. Removed.